### ISSUE TITLE
Where is Quick Action inside CSS to convert Image to Base64?

### ISSUE THREAD

**Issue by AchimStuy** [madskristensen/WebEssentials2019#12]
### Installed product versions
- Visual Studio: 2019 Professional
- This extension: 0.5.10

### Description
In former versions of Visual Studio and this extensions there was a quick action when editing a CSS-file to embed linked images as base64 dataUri. Where is it gone? 

### Steps to recreate
See https://www.hanselman.com/blog/useful-visual-studio-extension-web-essentials-from-mads-kristensen or https://www.itorian.com/2014/02/base64-encode-image-in-website-web.html

### Current behavior
There is no quick action menu.

### Expected behavior
A quick action menu providing me a command to convert url to base64.

**Comment by ahwm** [comment]
As a workaround, you can right-click the file and choose Copy base64 DataURI to clipboard and then paste it.

![image](https://user-images.githubusercontent.com/20478373/98852642-365e6080-2415-11eb-90a3-b73db57684ee.png)

![image](https://user-images.githubusercontent.com/20478373/98852727-53932f00-2415-11eb-9dd1-360967302e8c.png)

![image](https://user-images.githubusercontent.com/20478373/98852753-5ee65a80-2415-11eb-92a4-1521209f9b92.png)

**Comment by AchimStuy** [comment]
Yeah, I've seen that. But I really liked the toggle behavior inside CSS-files.
